testing/vcs/test_animate_isoline.py

```diff
@@ -0,0 +1,33 @@
+import vcs
+import cdms2
+import os
+import sys
+import time
+pth = os.path.join(os.path.dirname(__file__),"..")
+sys.path.append(pth)
+import checkimage
+
+f=cdms2.open(os.path.join(vcs.prefix,"sample_data","clt.nc"))
+s=f("clt",slice(0,12)) # read only 12 times steps to speed up things
+
+x=vcs.init()
+x.drawlogooff()
+x.setbgoutputdimensions(1200,1091,units="pixels")
+
+iso=x.createisoline()
+x.plot(s,iso,bg=1)
+x.animate.create()
+print "Saving now"
+prefix= os.path.split(__file__)[1][:-3]
+x.animate.save("%s.mp4"%prefix)
+pngs = x.animate.close(preserve_pngs = True) # so we can look at them again
```

Comment:
@doutriaux1 @chaosphere2112 Will this use a unique prefix for the png files?

Dashboards often run `ctest -jX` to speed things up by running test in parallel. If these animation tests would overwrite each other, we'll need to make sure that parallel testing is disabled.